src/App.tsx

```diff
@@ -100,6 +97,7 @@ const App = (props: AppProps) => {
   );
 
   function updateCycles(newCycles: Cycle[]) {
+    const maxOfCycles = getMaxStoredCountOfCycles(maxDisplayedCycles);
```

Comment:
You don't need `maxOfCycles` variable, because name of the function already describes what we'll getting here
Just use the function call as parameter below